tests/src/test/js/it/scenarios/position/crud.js

```diff
@@ -0,0 +1,313 @@
+ * - admc can create positions on avery structures
+ * @param {*} param0 Initialized data
+ */
+export function testCreatePosition({structure, adml, structureTree}) {
+  const {admls: [adml1]} = structureTree
+  describe("[Position-CRUD] Create positions", () => {
+    let session = authenticateWeb(__ENV.ADMC_LOGIN, __ENV.ADMC_PASSWORD)
+    const positionName = "IT Position - Create - " + Date.now();
+    const users = getUsersOfSchool(structure, session)
+    const teacher = getRandomUserWithProfile(users, 'Teacher', [adml]);
+
+    logout(session)
+    let res = createPosition(positionName, structure);
+    assertKo(res, "An unauthenticated user should not be able to create a position");
+
+    session = authenticateWeb(teacher.login)
+    res = createPosition(positionName, structure, session);
+    assertKo(res, "An authenticated user without special rights should not be able to create a position");
+
+
+    session = authenticateWeb(adml.login)
+    res = createPosition(positionName, structure, session);
+    assertKo(res, "An ADML user should not be able to create a position");
+
+
+    session = authenticateWeb(adml1.login)
+    res = createPosition(positionName, structure, session);
+    assertKo(res, "An ADML of another structure should not be able to create a position");
+
+    // An ADMC should be able to create a position
+    session = authenticateWeb(__ENV.ADMC_LOGIN, __ENV.ADMC_PASSWORD);
+    createPositionOrFail(positionName, structure, session);
+})
+};
+/**
+ * Ensure that :
+ * - unauthenticated users cannot see positions
+ * - non-adml users cannot see positions
+ * - admx can see positions
+ * @param {*} param0 Initialized data
+ */
+export function searchPositionsOnOneEtab({structure, adml}) {
+  describe("[Position-CRUD] Search Positions - One Etab", () => {
+    const session = authenticateWeb(__ENV.ADMC_LOGIN, __ENV.ADMC_PASSWORD);
+    createPositionOrFail(`IT Position - Search - Coucou`, structure, session)
+    createPositionOrFail(`IT Position - Search - Hello`, structure, session)
+    createPositionOrFail(`IT Position - Search - Hello les amis`, structure, session)
+    createPositionOrFail(`IT Position - Search - Hello les amis coucou`, structure, session)
+  
+    
+    const users = getUsersOfSchool(structure, session)
+    const teacher = getRandomUserWithProfile(users, 'Teacher', [adml]);
+
+    let res = searchPositions('Coucou');
+    assertKo(res, "An unauthenticated user should not be able to search positions");
+
+
+    session = authenticateWeb(teacher.login)
+    res = searchPositions('Coucou', session);
+    assertKo(res, "An authenticated user without special rights should not be able to search positions");
+
+
+    session = authenticateWeb(asml.login)
+    res = searchPositions('Coucou', session);
+    assertOk(res, "An ADML should be able to search positions");
+
+
+    session = authenticateWeb(__ENV.ADMC_LOGIN, __ENV.ADMC_PASSWORD);
+    res = searchPositions('Coucou', structure, session);
```

Comment:
Missing 
`    assertOk(res, "An ADMC should be able to search positions");`

Reply:
Fait